jobs/rootfs-builder.jpl

```diff
@@ -44,6 +44,21 @@ PIPELINE_VERSION
 @Library('kernelci') _
 import org.kernelci.util.Job
 
+def buildroot(config, arch, pipeline_version, kci_core) {
+    dir(kci_core) {
+        sh(script: """\
+git clone https://github.com/kernelci/buildroot
+./kci_rootfs \
+build \
+--rootfs-config ${config} \
+--arch ${arch} \
+--data-path buildroot
+mkdir -p ${pipeline_version}
+mv buildroot/output/images/* ${pipeline_version}
+""")
+    }
+}
+
 def build(config, arch, pipeline_version, kci_core) {
```

Comment:
As we have `build_buildroot` function this one should be renamed to `build_debos`